.github/workflows/publish.yaml

```diff
@@ -0,0 +1,34 @@
+name: Publish
+
+on:
+  push:
+    tags:
+      - '0.*'
+      - '1.*'
```

Comment:
Where do you sit on the "v or no v" question? GitHub's releases page says:

> It’s common practice to prefix your version names with the letter v. Some good tag names might be v1.0.0 or v2.3.4.

CPython, requests, Juju, numpy, Pydantic, use "v". uv, Django, Jinja, FastAPI, Go do not.

I'm not certain on this, but what I've heard is the original motivation of the "v" prefix was for tooling that was creating the artefacts, and that this is all out of date now.

I like the "no v" look better, but the prominence of the "use v" note in GitHub makes me wonder if that would be better.

Reply:
Good call. I actually like the prefix (Go modules uses it as well) in part because it's easier to filter on, and so easy to see "this is a version tag". Changed to that.